stytra/tests/test_init_gui.py

```diff
@@ -0,0 +1,66 @@
+from time import sleep, time
+
+from lightparam import Param
+import stytra
+
+from stytra.stimulation import Protocol, Pause
+from stytra.experiments import VisualExperiment
+from stytra.stimulation.stimuli import FullFieldVisualStimulus
+from stytra.triggering import Trigger
+from PyQt5.QtWidgets import QApplication
+from PyQt5.QtCore import Qt
+import stytra as st
+from pathlib import Path
+from pkgutil import iter_modules
+from importlib import import_module
+import pytest
+
+# iterate through the modules in the current package
+package_dir = Path(st.__file__).parent / "examples"
+
+protocols = []
+for (_, module_name, _) in iter_modules([package_dir]):
+
+    # import the module and iterate through its attributes
+    try:
+        module = import_module(f"stytra.examples.{module_name}")
+    except ImportError as e:
+        print("Error in: {}\nSee full message here:\n{}".format(module_name, e))
+
+    # check if external harware is required to run the example
+    if (
+        "REQUIRES_EXTERNAL_HARDWARE" in dir(module)
+        and getattr(module, "REQUIRES_EXTERNAL_HARDWARE") == True
+    ):
+        pass
+    else:
+        for attribute_name in dir(module):
+            if "Protocol" in attribute_name and attribute_name != "Protocol":
+                protocols.append(getattr(module, attribute_name))
+            # attribute = getattr(module, attribute_name)
+        # except ModuleNotFoundError:
+        #    print(f"Can't import {module}")
```

Comment:
commented out why?

Reply:
No reason,I fixed it [here](https://github.com/portugueslab/stytra/blob/f5847cce859cb3be0d3e601491b3b0775a7e2152/stytra/tests/test_init_gui.py#L27-L36) by addind the catch for module not found during the import

both in test_init_gui.py and test_examples.py